app/controllers/claimToAdjustPoa/SuccessController.scala

```diff
@@ -16,29 +16,60 @@
 
 package controllers.claimToAdjustPoa
 
+import cats.data.EitherT
 import config.{AgentItvcErrorHandler, FrontendAppConfig, ItvcErrorHandler}
 import controllers.agent.predicates.ClientConfirmedController
+import models.core.Nino
+import play.api.Logger
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.{ClaimToAdjustService, PaymentOnAccountSessionService}
 import uk.gov.hmrc.auth.core.AuthorisedFunctions
-import utils.AuthenticatorPredicate
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.{AuthenticatorPredicate, ClaimToAdjustUtils}
+import views.html.claimToAdjustPoa.SuccessView
 
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
 class SuccessController @Inject()(val authorisedFunctions: AuthorisedFunctions,
-                                  val auth: AuthenticatorPredicate,
+                                  val view: SuccessView,
```

Comment:
I think we need to rename this controller to something else, so to easy up reading this in the future

Reply:
Does "SuccessMessagePageController" sounds better?

Reply:
I think any reference to ClaimToAdjust sub-domain would nice to have, as its not clear from the name what this is related to

Reply:
Okay I see, so something like SuccessMessagePoaController or SuccessForPoaJourneyController I think is more suitable

Reply:
PaymentsOnAccountAdjustedController or PaymentsOnAccountUpdatedController, something similar would work